MobileClient/SignupScreen.js

```diff
@@ -0,0 +1,150 @@
+            </Text>
+            <View style={styles.action}><TextInput style={styles.input} placeholder={"First Name"} placeholderTextColor='grey' />
+            </View>
+            <View style={styles.action}><TextInput style={styles.input} placeholder={"Last Name"} placeholderTextColor='grey' />
+            </View>
+            <View style={styles.action}><TextInput style={styles.input} placeholder={"Email"} keyboardType={'email-address'} placeholderTextColor='grey' />
+            </View>
+            <View style={styles.action}><TextInput style={styles.input} placeholder={"Password"} secureTextEntry={entryP} placeholderTextColor='grey' />
+                <TouchableOpacity onPress={handlePress_Pass}>
+                    <Feather style={styles.icons}
+                        name={iconP}
+                        color="grey"
+                        size={20}
+                    />
+                </TouchableOpacity>
+
+
+            </View>
+            <View style={styles.action}><TextInput style={styles.input} placeholder={"Confirm Password"} placeholderTextColor='grey' secureTextEntry={entryC} />
+                <TouchableOpacity onPress={handlePress_Confirm}>
+                    <Feather style={styles.icons}
+                        name={iconC}
+                        color="grey"
+                        size={20}
+                    />
+                </TouchableOpacity>
+
+            </View>
+            <TouchableOpacity style={styles.signupButton}>
+                <Text style={styles.signupText}>
+                    CREATE ACCOUNT
+                </Text>
+            </TouchableOpacity>
+            {/* <TouchableOpacity style={styles.passwordContainer}>
+                <Text style={styles.passwordText}>
+
+                </Text>
+            </TouchableOpacity> */}
+            <TouchableOpacity style={styles.supporterButton}>
+                <Text style={styles.supporterText}>
+                    REQUEST SUPPORTER ACCOUNT
+                </Text>
+            </TouchableOpacity>
+        </View >
+    );
+}
+
+const styles = StyleSheet.create({
+
+    container: {
+        flex: 1,
+        padding: 20,
+        alignItems: "center",
+        justifyContent: "center",
+    },
+    signupButton: {
+        marginTop: 20,
+        marginBottom: 10,
+        padding: 18,
+        backgroundColor: 'maroon',
+        borderRadius: 8,
+        alignItems: "center",
+        justifyContent: "center",
+        width: "100%",
+        shadowColor: 'rgba(0,0,0, .8)', // IOS
+        shadowOffset: { height: 1, width: 1 }, // IOS
+        shadowOpacity: 1, // IOS
+        shadowRadius: 2, //IOS
+        elevation: 2 // Android
+    },
+    action: {
+        flexDirection: 'row',
+        marginTop: 10,
+        borderBottomWidth: 2,
+        borderBottomColor: '#f2f2f2',
+        paddingBottom: 5
+    },
+    icons: {
+        marginTop: 20
+    },
+    input: {
+        fontFamily: 'OpenSans-Light',
```

Comment:
Is there a way to define a default throughout the app for this font and the maroon color? I know on the web side we only need to define it once in index

Reply:
I am not sure, for now should I just store it in a variable?

Reply:
https://stackoverflow.com/questions/35255645/how-to-set-default-font-family-in-react-native

Reply:
Done!